src/provisioner/provisioner.py

```diff
@@ -206,7 +183,7 @@ def _get_node_info(cortx_conf: ConfigStore):
         CortxProvisionerLog.reinitialize(
             const.SERVICE_NAME, log_path, level=log_level)
 
-        if cortx_conf.get(f'node>{node_id}') is None:
+        if cortx_conf.get(f'node>{node_id}>name') is None:
```

Comment:
Error message needs to be correct. Since you are checking or name, it should say that.